@re-/model/demos/generateCode/mapTSfiles.ts

```diff
@@ -1,56 +1,133 @@
-import { fromHere } from "@re-/node"
-import { Project, Statement, ts } from "ts-morph"
+import { Project, SourceFile } from "ts-morph"
+import { Stack } from "../stack.js"
+import { addBlockToMaps, getComment } from "./helpers.js"
 import { regex } from "./regex.js"
 
-const { tagMatch, blockCommentMatch, commentMatch } = regex
+const {
```

Comment:
Normally I love destructuring everything, but in this case I'd just access these variables as props, (i.e. `regex.blockGrab`) as it helps clarify what they are without making it too clunky to use them.